.azurepipelines/Release.yml

```diff
@@ -17,35 +17,171 @@ resources:
       type: github
       endpoint: microsoft
       name: microsoft/mu_devops
-      ref: refs/tags/v2.5.2
+      ref: main
 
 name: $(Date:yyyyMMdd)$(Rev:.r) Mu Tiano Platforms Release
 
 stages:
-- stage: BuildQemuPlatforms
-  displayName: Build QEMU Platforms
+- stage: LinuxBuildQemuPlatforms
+  displayName: Build QEMU Platforms on Linux
+
+  variables:
+  - group: tool-chain-ubuntu-gcc
+
   jobs:
     - template: Platform-Build-Job.yml
       parameters:
-          extra_install_step:
-                - script: sudo dnf --assumeyes install openssl-devel
-                  displayName: Install openssl
-          container_image: ghcr.io/tianocore/containers/fedora-37-test:54e5bd1
-          tool_chain_tag: GCC5
-          vm_image: ubuntu-latest
+        tool_chain_tag: 'GCC5'
+        vm_image: ubuntu-latest
+        os_type: Linux
+        container_image: ghcr.io/tianocore/containers/fedora-37-test:latest

```

Comment:
Should we use the ubuntu ones?

Reply:
I was going to let @cfernald add that to #567

Reply:
Definitely not opposed to doing it here though if anyone would like that.